src/common/helpers/emp/transforms/manual-coordinates.js

```diff
@@ -0,0 +1,80 @@
+import proj4 from 'proj4'
+import './proj4-definition-osgb34.js'
+import { generateCirclePolygon } from './circle-to-polygon.js'
+import { COORDINATE_SYSTEMS } from '../../../constants/coordinates.js'
+
+const { OSGB36, WGS84 } = COORDINATE_SYSTEMS
+
+const singleOSGB36toWGS84 = ({ eastings, northings }) =>
+  proj4('OSGB36', 'WGS84', [
+    Number.parseFloat(eastings),
+    Number.parseFloat(northings)
+  ])
+
+const areCoordsTheSame = ([lon1, lat1], [lon2, lat2]) => {
+  return lon1 === lon2 && lat1 === lat2
+}
+
+export const manualCoordsToEmpGeometry = (siteDetails) => {
+  return siteDetails.map((site) => {
+    let coords
+    if (site.coordinatesEntry === 'single') {
+      coords = circleToEmp(site)
+    }
+    if (site.coordinatesEntry === 'multiple') {
+      coords = polygonToEmp(site)
+    }
+    if (coords) {
+      // For a polygon, the last point should be a copy of the first point
+      // https://developers.arcgis.com/rest/services-reference/enterprise/geometry-objects/#polygon
+      if (!areCoordsTheSame(coords[0], coords[coords.length - 1])) {
```

Comment:
```suggestion
      if (!areCoordsTheSame(coords[0], coords.at(-1)) {
```
thing.length -1 is no longer the way we do it.  Sonar complains. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at